### PR TITLE
Add matomo tag

### PR DESCRIPTION
**Pull request by liamocn**
## Related issues

#14 

## Checklist

- [x] **DO** Unit tests pass.
- [ ] **DO** If new feature, created unit test.

## Purpose

This adds the matomo tag to all of the portal pages, for tracking page views, etc

#### Does this introduce a major change?
- [ ] Yes
- [x] No

## Implementation Detail

The matomo server URL and site ID are configurable. They will default to `matomo-dev.acelab.ca` and `1` respectively, but can be set in the `.flaskenv` file. This is to distinguish production tracking from any other environment. See the flaskenv template for details.

### CODE REVIEW

File: flaskenv.template [hunk 1]
```diff
@@ -17,4 +17,6 @@ ORCID_OAUTH_CLIENT_SECRET=<ORCID SECRET ID>
 ORCID_TOKEN_URL=https://orcid.org/oauth/token
 ADMIN_EMAIL = <Your gmail>
 WEBHOOKS_SECRET=<github webhook secret>
+MATOMO_SERVER_URL=matomo-dev.acelab.ca
```
Comment (cmadjar) [inline flaskenv.template:20]: can you modify this to 
```suggestion
MATOMO_SERVER_URL=<your Matomo URL>
```

File: config.py [hunk 2]
```diff
@@ -65,6 +65,10 @@ class Config(object):
     # GitHub webhooks
     WEBHOOKS_SECRET = os.environ.get("WEBHOOKS_SECRET")
 
+    # Matomo
+    MATOMO_SERVER_URL = os.environ.get("MATOMO_SERVER_URL", "matomo-dev.acelab.ca")
```
Comment (cmadjar) [inline config.py:69]: can you modify this to

```suggestion
    MATOMO_SERVER_URL = os.environ.get("MATOMO_SERVER_URL", <your Matomo URL>)
```

**Review by cmadjar — CHANGES_REQUESTED** [review]
@liamocn two minor comments. Otherwise looks good.

File: config.py (hunk 2 above)
Comment (liamocn) [reply config.py]: The fallback value needs to be valid so I've removed it from here instead

**Review by cmadjar — APPROVED** [review]
LGTM

**Pull request merged**